ppl/zqd/db/postgresdb/ztests/tenant.yaml

```diff
@@ -0,0 +1,81 @@
+tag: postgres
```

Comment:
Point of discussion: As an outsider looking in, I find these story tests difficult to grok. When failures occur it is often  hard to find what caused the failure. I would prefer this be broken down into a suite of tests. Perhaps something like:
- auth/ls
- auth/rm
- auth/info
- auth/rename

Reply:
@mattnibs : if I split up the tests, there's a lot of boilerplate that would need to be duplicated between the separate test files - the auth zqd configuration, the zapi user setup, the creation of the spaces, and test input files. Since these operations are all to verify the space related apis, is there something else I could do to make the test easier to understand? Let me add some extra comments to one of the files, and see if that helps.

Reply:
Take a look at 252ce57168f7aab60e99b8acbb7265cee3099caa and let me know if that helps.